meta-mender-tegra/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -1 +1,22 @@
 RDEPENDS_${PN}_append_tegra = "${@' tegra-bup-payload libubootenv-fake' if d.getVar('PREFERRED_PROVIDER_virtual/bootloader').startswith('cboot') else ''}"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI += " \
+    file://mender \
+    file://mender-client-commit.service \
+    file://mender-client-commit.sh \
+"
+
+FILES_${PN} += "${systemd_system_unitdir}/mender-client-commit.service"
+FILES_${PN} += "${sysconfdir}/mender/mender-client-commit.sh"
+FILES_${PN} += "${bindir}/mender"
+
+SYSTEMD_SERVICE_${PN} = "mender-client-commit.service"
```

Comment:
@jajoosiddhant you need to change this to += so it doesn't replace the mender client service setup [here](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-mender/mender-client/mender-client.inc#L30)

Reply:
Right now I see:
```
systemctl status mender-client
Unit mender.service could not be found.
● mender-client.service - Mender OTA update service
     Loaded: loaded (/lib/systemd/system/mender-client.service; disabled; vendor preset: disabled)
     Active: inactive (dead)
```

Reply:
actually I think you should use `append_mender-uboot` here